Projects/MoveFlow/Sources/View/MovingFlowConfirmScreen.swift

```diff
@@ -29,16 +29,40 @@ struct MovingFlowConfirmScreen: View {
                         displayItems: quote.displayItems.map({ .init(title: $0.displayTitle, value: $0.displayValue) }
                         ),
                         insuranceLimits: quote.insurableLimits,
-                        typeOfContract: quote.contractType,
-                        onInfoClick: quote.quoteInfo != nil
-                            ? {
-                                movingFlowNavigationVm.isInfoViewPresented = quote.quoteInfo
-                            } : nil
+                        typeOfContract: quote.contractType
                     )
                 })
 
+            let _ = getQuotes(from: movingFlowModel)
```

Comment:
Can we call getQuotes(from: movingFlowModel) once and reuse it in 2 places instead of calling it 2 times ?